x-pack/legacy/plugins/rollup/public/crud_app/sections/job_create/steps/step_review.js

```diff
@@ -122,12 +127,25 @@ export class StepReviewUi extends Component {
         {this.renderTabs()}
 
         <EuiErrorBoundary>
-          <JobDetails
-            job={job}
-            json={json}
-            tab={selectedTab}
-          />
+          <JobDetails job={job} json={json} tab={selectedTab} />
         </EuiErrorBoundary>
+
+        <EuiSpacer size="m" />
+
+        <EuiCheckbox
```

Comment:
Feel free to ping the Kibana Design team for some input from a designer on where this checkbox should go within this layout. Personally I think it works where it is but I have a feeling there might be a better place for it.

Reply:
+1 on design feedback. I think it might look nice to the immediate right of the Save button

cc @elastic/kibana-design